### ISSUE TITLE
Error creating new template from file: template: .DS_Store:5: unexpected "," in operand

### ISSUE THREAD

**Issue by ericglau** [appsody/appsody#674]
**Describe the bug**
In Appsody 0.5.0, some arbitrary files in the stack directory can cause stack package to encounter an error.  For example on macOS, if you use Finder to browse to the stack directory, it creates a hidden .DS_Store file.  Then when you run `appsody stack package`, the error below occurs.

**To Reproduce**
1. Clone the `dsstore` branch of https://github.com/ericglau/interactive-stack: `git clone -b dsstore https://github.com/ericglau/interactive-stack.git` which has a .DS_Store file in the root directory
2. Run `appsody stack package`

**Expected behavior**
The stack should be packaged.

**Actual behaviour**
An error occurs:
```
******************************************
Running appsody stack package
******************************************
Running git command: git version
Running git command: git status -sb
Attempting to perform git config --local remote.<origin>.url  ...
Running git command: git config --local remote.origin.url
Running git command: git log -n 1 --pretty=format:'{"author":"%an", "authoremail":"%ae", "sha":"%H", "date":"%cd", "committer":"%cn", "committeremail":"%ce", "message":"%s"}'
[Error] Error applying templating: Error walking through directory: Error creating new template from file: template: .DS_Store:5: unexpected "," in operand
```
If you delete the .DS_Store file, the error does not occur.

**Environment Details (please complete the following information):**
 - OS: macOS Catalina 10.15.1

**If applicable please specify:**
 - CLI version: Appsody 0.5.0
 - Stack you are using: https://github.com/ericglau/interactive-stack/tree/dsstore

**Additional context**
The error does not occur with Appsody 0.4.10 even if the .DS_Store file is there.

**Comment by kylegc** [comment]
@tnixa please take a look at this

**Comment by Kamran64** [comment]
FYI: I think @CameronMcWilliam is looking at this.

**Comment by CameronMcWilliam** [comment]
Hi  @ericglau this has been fixed in #634 and will be included in the next release of Appsody.
